containers/buildah
# internal/mkcw/embed/entrypoint.gz: compress with -n9

#### What type of PR is this?

/kind other

#### What this PR does / why we need it:

Compress the unencrypted entry point binary for confidential workload images using gzip's `-n` flag, to omit the original file's timestamp from the compressed copy.

#### How to verify it

Tests should continue to pass, and we should see the newly-renamed `internal/mkcw/embed/entrypoint_amd64.gz` being updated less often in other pull requests.

#### Which issue(s) this PR fixes:

Should fix https://github.com/containers/buildah/issues/5191.

#### Special notes for your reviewer:

#### Does this PR introduce a user-facing change?

```release-note
None
```
